src/components/organisms/Nav.js

```diff
@@ -72,6 +120,7 @@ const Nav = () => {
           clickHandler={() => setModalState(modalStates.signup)}
           variant="primary"
           text="Sign Up"
+          disabled={isDisabled}
```

Comment:
the button component doesn't have a "disabled" prop. not sure what this is meant to do for the signup button in the nav.